spec/swagger.yaml

```diff
@@ -5616,6 +5616,53 @@ paths:
                     details: null
         429: *Error429ResponseDefinition
         500: *Error500ResponseDefinition
+  /jobs-api/v3/projects/{projectId}/jobs/{translationJobUid}/locales-completion-dates:
+    get:
+      summary: "Get last completion dates per locale of job"
+      tags:
+      - Jobs
+      description: "This get last completion dates per locale of job"
```

Comment:
Does the endpoint support `closed`/`deleted` jobs? Do we need to indicate such information in api documentation?

Reply:
yes, we do not have any limitation. Even locale was deleted from a job, this endpoint will return information for this locale.